test/orchestrator/workflow-runner-controller.test.ts

```
@@ -0,0 +1,886 @@
+    );
+
+    findWorkflowRunnerCommandReceipt.mockResolvedValueOnce({
+      commandKind: "set-state",
+      request: message.payload.command,
+      response: { trackingCommentId: 42 },
+    });
+    handleWorkflowRunnerMessage(socket as never, message);
+    await waitUntil(
+      () =>
+        socket.messages.filter(
+          (entry) => (entry as { type?: string }).type === "workflow-runner:command-result",
```

ESLint

[error] Too many nested callbacks (4). Maximum allowed is 3.

(max-nested-callbacks)